devicetypes/smartthings/zigbee-window-shade.src/zigbee-window-shade.groovy

```diff
@@ -145,5 +184,12 @@ def configure() {
     log.info "configure()"
     sendEvent(name: "checkInterval", value: 2 * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
     log.debug "Configuring Reporting and Bindings."
-    zigbee.configureReporting(CLUSTER_WINDOW_COVERING, ATTRIBUTE_POSITION_LIFT, DataType.UINT8, 0, 600, null) + refresh()
-}
\ No newline at end of file
+
+    def cmds
+    if (usesWindowCovering()) {
```

Comment:
I think this can now be replaced with `supportsLiftPercentage`

Reply:
@tpmanley I have changed it, please review